docs/ci.md

```diff
@@ -0,0 +1 @@
+ci/README.md
```

Comment:
This link is broken :
```
$ ls -l docs/ci.md 
lrwxrwxrwx. 1 greg greg 12 Feb  1 17:09 docs/ci.md -> ci/README.md
$ realpath docs/ci.md 
realpath: docs/ci.md: No such file or directory
```

Link target should be `../ci/README.md`.

Reply:
fixed. thanks